src/sass/themes.scss

```diff
@@ -0,0 +1,45 @@
+/* This file is not loaded, it is purely as an example for overwriting the variables to use a theme */
```

Comment:
Hier moet er toch een eenvoudigere oplossing voor zijn? Het inladen van 1 van de 2 bestanden (een theme dus) die bepaald of het donker of licht is?

Reply:
Ik zou dit dan gwn kopiëren in de project css. Normaal verander je daar zelf variabelen, maar als je dit wil nemen kan je het ovenemen. Zoals de components voorbeelden zijn van html, is dit voorbeeld van css. Of vind je dat dat anders moet kunnen?